yb-voyager/src/query/queryissue/parser_issue_detector_test.go

```diff
@@ -304,3 +304,19 @@ func TestUnloggedTableIssueReportedInOlderVersion(t *testing.T) {
 	assert.Equal(t, 1, len(issues))
 	assert.True(t, cmp.Equal(issues[0], NewUnloggedTableIssue("TABLE", "tbl_unlog", stmt)))
 }
+
+func TestSingleXMLIssueIsDetected(t *testing.T) {
```

Comment:
nit: small comment explaining the test that the stmt has two xml function/expression but should be reported only once.